graph-service/enricher.py

```
@@ -0,0 +1,875 @@
+        for event in event_elements:
+            try:
+                # Extract data attributes
+                data_uri = event.get('data-uri', '')
+                data_id = event.get('data-id', '')
+                data_score = event.get('data-score', '0')
+                
+                # Try to extract data-details JSON if available
+                data_details_str = event.get('data-details', '{}')
+                try:
+                    data_details = json.loads(data_details_str.replace('&quot;', '"'))
+                except:
```

Ruff (0.14.7)

Do not use bare `except`

(E722)